redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/incidentImageCarousel.test.js

```diff
@@ -17,6 +30,47 @@ describe('c-incident-image-carousel', () => {
         document.body.appendChild(element);
 
         const labelElement = element.shadowRoot.querySelector('p');
-        expect(labelElement.textContent).toContain('no images');
+        expect(labelElement.textContent).toContain(
+            'There are currently no images of damage for this case.'
+        );
+    });
+
+    it('renders two pictures when the wire mock returns two', () => {
+        // Create initial element
+        const element = createElement('c-incident-image-carousel', {
+            is: IncidentImageCarousel
+        });
+        document.body.appendChild(element);
+
+        // Emit data from @wire
+        getRecordAdapter.emit('mockRecordId');
+        getRelatedPicturesAdapter.emit(mockTwoImages);
+
+        return Promise.resolve().then(() => {
+            const imageEls = element.shadowRoot.querySelectorAll(
+                'lightning-carousel-image'
+            );
+            expect(imageEls.length).toEqual(mockTwoImages.length);
```

Comment:
We should also test that the `key` and `src` attributes get the value of url passed.

Reply:
src is now being matched.